docs/zh/latest/FAQ.md

```diff
@@ -566,3 +566,24 @@ apisix:
 
  - 插件实例作用范围：`plugin-metadata` 作用于该插件的所有配置实例。`plugin-configs` 作用于其下配置的插件配置实例。
  - 绑定主体作用范围：`plugin-metadata` 作用于该插件的所有配置实例绑定的主体。`plugin-configs` 作用于绑定了该 `plugin-configs` 的路由。
+
+## 如何修改APISIX的时区
+
+你可能需要将 APISIX 的时区设置为与你本地时区一致，否则在检索 APISIX 日志的时候会得到不一致的时区的日志数据，特别在你使用 APISIX 日志插件的时候。
```

Comment:
Hi, just to make sure: is it's necessary to use the same timezone as local? If yes, then please use `必须` or `应当` instead of `可能`.

Reply:
> Hi, just to make sure: is it's necessary to use the same timezone as local? If yes, then please use `必须` or `应当` instead of `可能`.

@juzhiyuan 
I think it depends on the user's actual situation. After all, in the current APISIX Docker image, the default time zone is UTC, but our own scene needs to use the same timezone as local. To make it easier to view the logs.

So I think `可能` is OK.

Reply:
Just a little confusing for me, cc @bzp2010 to have a check. 😅